site/zenodo_rdm/moderation/config.py

```diff
@@ -46,12 +52,14 @@
     links_rule,
     files_rule,
     text_sanitization_rule,
+    match_query_rule,
 ]
 """Scoring rules for record moderation."""
 
 MODERATION_COMMUNITY_SCORE_RULES = [
     links_rule,
     text_sanitization_rule,
     verified_user_rule,
+    match_query_rule,
```

Comment:
minor: I think for communities we'll need a different rule/config, since the mappings are different